settings.lua

```diff
@@ -30,4 +30,14 @@ data:extend{
     minimum_value = 1,
     default_value = 2500,
   },
-}
\ No newline at end of file
+
+  {
```

Comment:
Indentation.

Reply:
Ah, whoops. Must've indented improperly when I copied this over. I didn't make a localized key for the setting. Forgot to do that.